pkg/controller/v1alpha2/core/scopes/healthscope/healthscope_controller.go

```diff
@@ -48,10 +46,13 @@ const (
 // Reconcile error strings.
 const (
 	errGetHealthScope          = "cannot get health scope"
-	errMarshalScopeDiagnosiis  = "cannot marshal diagnosis of the scope"
 	errUpdateHealthScopeStatus = "cannot update health scope status"
 )
 
+const (
+	infoFmtScopeDiagnosis = "total workloads: %d, healthy workloads: %d, unhealthy workloads: %d, unknown workloads: %d"
```

Comment:
I suggest we can also provide a structural way here, makes it easy for other traits re-use this info.

Reply:
got it.